src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/LanguageServerRequestBaseParams.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    /// <summary>
+    /// Base class to represent the request params.
+    /// </summary>
+    public class LanguageServerRequestBaseParams
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Can this be kept internal?

Reply:
I would love to do that but i have no other option but to make it public. The children of this class are public and in use in the backend code of power apps cards. I cannot have an internal parent class and public children. I get compiler errors. As much as i love to, I cannot make the children internal as well since they are used outside of PowerFx. We should work with appropriate owners of cards code to remove this dependency